pkg/build/strategies/sti/sti.go

```diff
@@ -533,7 +533,7 @@ func (builder *STI) Execute(command string, user string, config *api.Config) err
 		close(injectionComplete)
 	}
 
-	wg := sync.WaitGroup{}
+	var wg sync.WaitGroup
```

Comment:
We don't need to initialize WaitGroups, just declare them and they are ready for use.
Idiomatic example: https://golang.org/pkg/sync/#example_WaitGroup